usr/local/www/fbegin.inc

```diff
@@ -245,11 +256,12 @@ $gold_menu = array();
 $gold_menu[] = array(gettext("pfSense Gold"), "https://www.pfsense.org/gold");
 $gold_menu = msort(array_merge($gold_menu, return_ext_menu("Gold")),0);
 
-if(! $g['disablehelpmenu']) {
+if (!$g['disablehelpmenu']) {
 	$help_menu = array();
 	$help_menu[] = array(gettext("About this Page"), $helpurl);
-	if($g['product_name'] == "pfSense")
-	$help_menu[] = array(gettext("Bug Database"), "https://www.pfsense.org/j.php?jumpto=redmine");
+	if ($g['product_name'] == "pfSense") {
+		$help_menu[] = array(gettext("Bug Database"), "https://www.pfsense.org/j.php?jumpto=redmine");
+	}
```

Comment:
Note: this "if" statement was not tabbed correctly. I put the curlies and tabbing to make it clear what it is doing, without changing the current behavior..
But there are more things on the help menu, like User Forum... directly below this "if" test - when product_name is not pfSense, should they be in the Help menu or not?